packages/fxa-payments-server/src/routes/Checkout/index.test.tsx

```diff
@@ -347,15 +403,21 @@ describe('routes/Checkout', () => {
         usePaypalUIByDefault: true,
       },
     });
-
-    const fillOutZeForm = async () => {
+    const fillOutZeForm = async (shouldSubscribeToNewsletter = false) => {
       const { getByTestId } = screen;
       fireEvent.change(getByTestId('new-user-email'), {
         target: { value: newAccountEmail },
       });
       fireEvent.change(getByTestId('new-user-confirm-email'), {
         target: { value: newAccountEmail },
       });
+      if (shouldSubscribeToNewsletter) {
+        const newsletterCheckbox = getByTestId(
+          'new-user-subscribe-product-updates'
+        );
+        fireEvent.click(newsletterCheckbox);
+        expect(newsletterCheckbox.checked).toEqual(true);
+      }
```

Comment:
Do we need this?  It's the same test as the one above, and the checkbox is independent of the payment type.